### ISSUE TITLE
Loss becomes NaN at an early stage when trained with the latest version of lycoris.

### ISSUE THREAD

**Issue by StupidGame** [KohakuBlueleaf/LyCORIS#136]
If you train with the latest version of lycoris, the loss will be NaN at an early stage.

Screenshot of wandb
![2023-12-15](https://github.com/KohakuBlueleaf/LyCORIS/assets/64509243/50220143-3363-43d8-9c28-3b70f24d47ed)

**Comment by StupidGame** [comment]
`!accelerate launch --num_cpu_threads_per_process 12 sd-scripts/train_network.py \
  --pretrained_model_name_or_path=$pretrained_model_name_or_path \
  --dataset_config=$datasets_path \
  --network_dim=$network_dim \
  --network_alpha=$network_alpha \
  --output_dir=$output_dir \
  --lr_scheduler="cosine_with_restarts" \
  --lr_scheduler_num_cycles=2 \
  --text_encoder_lr=$text_lr \
  --unet_lr=$unet_lr \
  --output_name=$dream_booth_new_model \
  --prior_loss_weight=1.0 \
  --seed=42 \
  --max_train_epochs=$dream_booth_epochs \
  --optimizer_type="Lion" \
  --optimizer_args "weight_decay=1e-1" "betas=0.9,0.99" \
  --max_grad_norm=1.0 \
  --mixed_precision='fp16'\
  --xformers \
  --gradient_checkpointing \
  --save_precision='fp16' \
  --sample_every_n_epochs=1 \
  --sample_prompts=$prompts_path \
  --save_model_as=$dream_booth_model_ext \
  --cache_latents \
  --bucket_no_upscale \
  --log_with="wandb" \
  --wandb_api_key="a156eaf006b4bb38f565d4938684c66f0da523fa" \
  --network_module=lycoris.kohya \
  --network_args $conv_dim $conv_alpha "algo=full" \
  --max_token_length=150 \
  --logging_dir=logs \
  --noise_offset=0.05 \
  --adaptive_noise_scale=0.05 \
  --clip_skip=2 \
  --min_snr_gamma=5`
  
It's no different in LOHA. dim is 8, alpha is 32, conv is the same for each

**Comment by KohakuBlueleaf** [comment]
Use alpha=1
or way more lower lr.

**Comment by StupidGame** [comment]
thanks!
